plugins/base/src/main/resources/dokka/docs/kdoc/EnumEntries.kt.template

```diff
@@ -0,0 +1,3 @@
+Returns a representation of an immutable list of all enum entries, in the order they're declared.
+
+This method may be used to iterate over the enum entries.
```

Comment:
I didn't find the documentation text in the KEEP, and `entries` hasn't been documented anywhere else, so I decided to play it safe and stick to the short description similar to the `values()` function for now.

Should be discussed with @qwwdfsad and the libraries team, maybe there's a KDoc lying around somewhere, or the authors know how they want to document it.